src/qz/ws/HttpAboutServlet.java

```diff
@@ -76,6 +88,48 @@ private void generateJsonResponse(HttpServletRequest request, HttpServletRespons
         }
     }
 
+    private void generateCertResponse(HttpServletRequest request, HttpServletResponse response) {
+        try {
+            String alias = request.getServletPath().split("/")[2];
+            String certData = loadCertificate(alias);
+
+            if (certData != null) {
+                response.setStatus(HttpServletResponse.SC_OK);
+                response.setContentType("application/x-x509-ca-cert");
+
+                response.getOutputStream().print(certData);
+            } else {
+                response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+                response.getOutputStream().print("Could not find certificate with alias \"" + alias + "\" to download.");
+            }
+        }
+        catch(Exception e) {
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+            log.warn("Exception occurred loading certificate: {}", e.getMessage());
+        }
+    }
+
+    private String loadCertificate(String alias) throws GeneralSecurityException, IOException {
+        Properties sslProps = DeployUtilities.loadTrayProperties();
+
+        if (sslProps != null) {
+            KeyStore jks = KeyStore.getInstance("jks");
+            jks.load(new FileInputStream(new File(sslProps.getProperty("wss.keystore"))), sslProps.getProperty("wss.storepass").toCharArray());
+
+            if (jks.isCertificateEntry(alias)) {
+                Certificate cert = jks.getCertificate(alias);
+
+                return "-----BEGIN CERTIFICATE-----" + System.lineSeparator() +
```

Comment:
@bberenz did you mean to leave in this `System.lineSeparator()`?

Reply:
Nope, must of missed resetting it when I reverted a few changes.